packages/code-studio/src/main/AppMainContainer.tsx

```diff
@@ -1002,7 +1002,12 @@ export class AppMainContainer extends Component<
   }
 }
 
-const mapStateToProps = (state: RootState) => ({
+const mapStateToProps = (
+  state: RootState
+): Omit<
+  AppMainContainerProps,
+  'match' | 'setActiveTool' | 'updateDashboardData' | 'updateWorkspaceData'
+> => ({
```

Comment:
Another way to separate the props is in Linker.tsx. I kind of like the way Linker is done, what do you think? Doesn't need to be changed in this PR.

Reply:
I'm a fan of this approach. Wish I'd noticed it before. I won't address on this PR, as there were a bunch of them.